src/Okta.Sdk/Generated/Factor.Generated.cs

```diff
@@ -93,8 +100,8 @@ public Task<IFactor> ActivateAsync(VerifyFactorRequest verifyFactorRequest, stri
             => GetClient().UserFactors.ActivateFactorAsync(verifyFactorRequest, userId, Id, cancellationToken);
         
         /// <inheritdoc />
-        public Task<IVerifyFactorResponse> VerifyAsync(VerifyFactorRequest verifyFactorRequest, string userId, string templateId = null, CancellationToken cancellationToken = default(CancellationToken))
-            => GetClient().UserFactors.VerifyFactorAsync(verifyFactorRequest, userId, Id, templateId, cancellationToken);
+        public Task<IVerifyFactorResponse> VerifyAsync(VerifyFactorRequest verifyFactorRequest, string userId, string templateId = null, int? tokenLifetimeSeconds = 300, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Technically a breaking change to this method signature. This is fine since we're still in the beta stage, but we'll both need to keep an eye on this once we are stable.